test/com/facebook/buck/features/go/testdata/go_test/testOutput/BUCK.fixture

```diff
@@ -0,0 +1,5 @@
+go_test(
+  name='all_tests',
```

Comment:
Could you indent these lines with 4 spaces. Our formatter complains and will prevent checkin in our internal repo.